modules/cnv-about-the-vm-dashboard.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-viewing-information-about-vm-workloads.adoc
+
+[id="cnv-about-the-vm-dashboard_{context}"]
+= About the Virtual Machines dashboard
+
+Access the *Virtual Machines* dashboard from the {product-title} web console by navigating to the *Workloads* -> *Virtual Machines* page and
+clicking the name of a virtual machine.
+
+The dashboard includes the following cards:
+
+* *Details* provides identifying information about the virtual machine, including:
+** the name of the virtual machine
+** the namespace
+** when the virtual machine was created
```

Comment:
Nit: Would prefer this to be 'date' or 'time' of virtual machine creation

Reply:
Good point. I made the whole section more minimalistic and it's now just 'Date of creation'